naive_bayes/nb_author_id.py

```diff
@@ -22,12 +26,27 @@
 features_train, features_test, labels_train, labels_test = preprocess()
 
 
+def fit_and_predict(clf, features_train, features_test, labels_train, labels_test):
```

Comment:
This function was in `tools/mylib.py`, I suggest removing the `mylib.py` file as it can be used to solve the tasks!